assets/windows/installer.iss

```diff
@@ -167,4 +169,10 @@ begin
             end;
         end;
     end;
-end;
\ No newline at end of file
+end;
+
+[Registry]
+Root: HKA; Subkey: "Software\Classes\{#UriScheme}"; ValueType: "string"; ValueData: "URL:Custom Protocol"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\{#UriScheme}"; ValueType: "string"; ValueName: "URL Protocol"; ValueData: ""
+Root: HKA; Subkey: "Software\Classes\{#UriScheme}\DefaultIcon"; ValueType: "string"; ValueData: "{app}\{cm:AppName}.exe,0"
+Root: HKA; Subkey: "Software\Classes\{#UriScheme}\shell\open\command"; ValueType: "string"; ValueData: """{app}\{cm:AppName}.exe"" ""%1"""

```

Comment:
I don't know if it's clearer to put the UriScheme directly into the file (the same for Linux, Android, etc.).

We did as constant in the Blackbox for file extensions for example:
https://github.com/betaflight/blackbox-log-viewer/blob/8419b1b815d74602c8cf92eb0439aacda1d960d8/assets/windows/installer.iss#L37-L43

Until now we only injected some values in the file for Android because they depending on the version. This seems more a "fixed" thing.

I see it reads it from the package.json. Is a standard tag of package.json? I can't find it in the documentation.
It's easier to read if we put it directly in the different files, but if we decide to change the value we will need to do a "search", but I suppose this will not happen. I don't know what think others. Opinions?

Reply:
I definitely agree that the dynamic injection of the URI scheme might have been overengineered. The scheme will likely never change - especially if/once it is used in e.g. the ELRS web UI. I will update the MR unless anyone has any objections.

As for the `package.json` - indeed the newly added `uriScheme` is a custom/non-standardized field.

Reply:
PR updated - we no longer do dynamic substitutions for the `uriScheme`.